@NeusiaHilario 

Agora tenho este erro: penso que tem haver com indexes de strings no ficheiro de paramentos de conexao:
o meu ficheiro esta assim: [https://www.dropbox.com/s/9242j0f5dhmfxyu/credenciaisConn.txt?dl=0](url)


Caused by: java.lang.StringIndexOutOfBoundsException: String index out of range: -2
	at java.lang.String.substring(String.java:1931)
	at DataAcessLayer.conexao.<init>(conexao.java:116)
	at DataAcessLayer.UsuarioDAO.<init>(UsuarioDAO.java:22)
	at Service.UsuarioServicos.<init>(UsuarioServicos.java:25)
	at controller.LoginPage1Controller.loginButtonOnAction(LoginPage1Controller.java:84)

--- a/src/DataAcessLayer/conexao.java
+++ b/src/DataAcessLayer/conexao.java
@@ -114,7 +114,7 @@ import javax.swing.JOptionPane;
                host = credenciaisConn.get(0).substring(5);
               port = credenciaisConn.get(1).substring(5);
                 bd = credenciaisConn.get(2).substring(3);
-               user = credenciaisConn.get(3).substring(5);
+               user = credenciaisConn.get(3).substring(9);
                password= credenciaisConn.get(4).substring(9);
                
                 if (host.isEmpty()) {

